esp-wifi/src/wifi/event.rs

```diff
@@ -0,0 +1,52 @@
+use core::cell::RefCell;
+
+use critical_section::Mutex;
+
+mod sealed {
+    pub trait Sealed {}
+}
```

Comment:
Please move this to `esp_wifi::private::Sealed`. We're adding this trait there in a different PR and we don't need two.

Reply:
I was going to move the `get_handler` method into the `Sealed` trait so it can't be called by user-code and can be changed later. Should I still move the sealed trait?